<?xml version="1.0"?>
<launch>

    <include file="$(find probot_gazebo)/launch/probot_anno_with_gripper/probot_anno_with_gripper_bringup_moveit.launch" />

    <node pkg="probot_grasping" name="graspingDemo" type="graspingDemo" output="screen">
        <param name="table_length"          value="0.3"/>
        <param name="table_breadth"         value="0.3"/>
        <param name="pregrasp_x"            value="0.20"/>
        <param name="pregrasp_y"            value="-0.17"/>
        <param name="pregrasp_z"            value="0.28"/>
         <param name="grasp_x"            value="0.10"/>
         <param name="grasp_y"            value="0.06"/>
         <param name="hor_grasp_z"            value="0.054"/>
         <param name="ver_grasp_z"            value="0.035"/>
         <param name="hor_ratio"            value="0.1"/>
         <param name="ver_ratio"            value="0.1"/>
         <param name="change_angle"            value="0"/>

         
    </node>

</launch>
